Cabal/Cabal.cabal

```diff
@@ -342,6 +342,14 @@ library
   if !impl(ghc >= 7.0)
     default-extensions: CPP
 
+executable cabal-setup
```

Comment:
This means that `cabal install Cabal` will now also install `cabal-setup`. Would be nice to avoid that, perhaps by building `cabal-setup` only when a flag is enabled.